keyvault/client/keys/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Release History
 
 ## 4.0.0-preview.1 (2019-06-28)
-For release notes and more information please visit 
-https://aka.ms/azure-sdk-preview1-java
+Version 4.0.0-preview.1 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide our efforts can be found in the [Azure SDK Design Guidelines for
+.Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
+
+For release notes and more information please visit https://aka.ms/azure-sdk-preview1-java
```

Comment:
Make this a hyperlink that is human friendly, e.g. 
```suggestion
For details on the Azure SDK for Java (July 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview1-java).
```

Reply:
updated.